erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -165,6 +165,9 @@ def get_conditions(filters):
 	if filters.get("voucher_no"):
 		conditions.append("voucher_no=%(voucher_no)s")
 
+	if filters.get("against_voucher"):
+		conditions.append("against_voucher=%(against_voucher)s")
+
```

Comment:
Please add docstrings and comments to this function. It will enhance the quality of our codebase